docs/data/sql_connectors.yml

```diff
@@ -48,6 +48,12 @@ avro-confluent:
     category: format
     sql_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-avro-confluent-registry/$version/flink-sql-avro-confluent-registry-$version.jar
 
+avro-glue:
+    name: Avro AWS Glue Schema Registry
+    maven: flink-avro-glue-schema-registry
+    category: format
+    sql_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-avro-glue-schema-registry/$version/flink-avro-glue-schema-registry-$version.jar
```

Comment:
I believe this needs to include the `$scala_version`, and if possible, only render for Scala 2.12

Reply:
The module is pure java and the only dependencies that it has that are scala-binary-versioned are `test` or `provided` scope.  The module is currently located [here](https://repo1.maven.org/maven2/org/apache/flink/flink-avro-glue-schema-registry/) in the central repository, so I don't think this is the case.

Reply:
This has recently changed, since 1.15 it will only support Scala 2.12. You can see the new artifact [here](https://repository.apache.org/content/repositories/snapshots/org/apache/flink/flink-avro-glue-schema-registry_2.12/1.15-SNAPSHOT/).